data/common.yaml

```diff
@@ -0,0 +1,28 @@
+rkhunter::allowhiddenfile:
+  - '/usr/lib*/hmaccalc/sha384hmac.hmac'
+  - '/usr/lib*/hmaccalc/sha512hmac.hmac'
+  - '/usr/sbin/.ispec.hmac'
+  - '/usr/sbin/.sshd.hmac'
+  - '/etc/.updated'
+  - '/usr/share/man/man1/..1.gz'
+  - '/usr/share/man/man5/.k5login.5.gz'
+  - '/usr/share/man/man5/.k5identity.5.gz'
+rkhunter::dbdir: '/var/lib/rkhunter/db'
+rkhunter::disable_tests: 
+  - 'suspscan hidden_ports deleted_files packet_cap_apps apps ipc_shared_mem'
```

Comment:
This is a suggested list of tests to disable by default because they are io intensive and/or they result in a lot of false positives. They should only be run when you are specifically wanting to run them.

Reply:
and yea, I got annoyed by how i had stuff laid out before, and beat yah to the punch on splitting it up into array.